### PR TITLE
Fix multi-line strings in DeclareLaunchArgument

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
I deleted my fork. I forgot https://github.com/UniversalRobots/Universal_Robots_ROS2_Ignition_Simulation/pull/16 was not merged yet. I restored the branch with my changes.

### CODE REVIEW

**Comment by fmauch** [comment]
@MatthijsBurgh If you could fix the merge conflicts introduced by #7 that would be awesome!

**Comment by MatthijsBurgh** [comment]
@fmauch please have a look at https://github.com/UniversalRobots/Universal_Robots_ROS2_GZ_Simulation/pull/18/files#diff-07ff1e87117ca592c12e8dcd194c3721e5cd805507f87a755e66fb65fce40670L72 and https://github.com/UniversalRobots/Universal_Robots_ROS2_GZ_Simulation/pull/18/files#diff-07ff1e87117ca592c12e8dcd194c3721e5cd805507f87a755e66fb65fce40670L80. As IMO the URDF/XACRO file should be in the description package. The description package should be configurable, to make the launch file reusable and also because you use these arguments in `ur_sim_moveit.launch.py`.

But my changes create a conflict for the default values. So please let me know how you want to solve it.

File: ur_simulation_gz/launch/ur_sim_control.launch.py [hunk 1]
```diff
@@ -58,6 +58,8 @@ def launch_setup(context, *args, **kwargs):
     # General arguments
     runtime_config_package = LaunchConfiguration("runtime_config_package")
     controllers_file = LaunchConfiguration("controllers_file")
+    description_package = LaunchConfiguration("description_package")
+    description_file = LaunchConfiguration("description_file")
```
Comment (fmauch) [inline ur_simulation_gz/launch/ur_sim_control.launch.py]: please revert that change, as we want to get rid of the "description package" structure.
```suggestion
```

File: ur_simulation_gz/launch/ur_sim_control.launch.py [hunk 2]
```diff
@@ -69,15 +71,15 @@ def launch_setup(context, *args, **kwargs):
     )
 
     rviz_config_file = PathJoinSubstitution(
-        [FindPackageShare("ur_description"), "rviz", "view_robot.rviz"]
+        [FindPackageShare(description_package), "rviz", "view_robot.rviz"]
```
Comment (fmauch) [inline ur_simulation_gz/launch/ur_sim_control.launch.py]: please revert that change, as we want to get rid of the "description package" structure.

File: ur_simulation_gz/launch/ur_sim_control.launch.py [hunk 3]
```diff
@@ -69,15 +71,15 @@ def launch_setup(context, *args, **kwargs):
...
     )
 
     robot_description_content = Command(
         [
             PathJoinSubstitution([FindExecutable(name="xacro")]),
             " ",
             PathJoinSubstitution(
-                [FindPackageShare("ur_simulation_gz"), "urdf", "ur_gz.urdf.xacro"]
+                [FindPackageShare(description_package), "urdf", description_file]
```
Comment (fmauch) [inline ur_simulation_gz/launch/ur_sim_control.launch.py]: That change is not required, it's supposed to be like that.

File: ur_simulation_gz/launch/ur_sim_control.launch.py [hunk 4]
```diff
@@ -98,6 +100,8 @@ def launch_setup(context, *args, **kwargs):
             "prefix:=",
             prefix,
             " ",
+            "sim_ignition:=true",
+            " ",
```
Comment (fmauch) [inline ur_simulation_gz/launch/ur_sim_control.launch.py]: That's not needed anymore.
```suggestion
```

**Comment by fmauch** [comment]
@MatthijsBurgh that's exactly one of the changes we did in the latest refactoring of the description. The description now only contains the actual kinematic description. The ros2_control part gets added in the places that actually start the control structure. Hence, every control package (driver, gazebo classic, gz sim) build their [own description](https://github.com/UniversalRobots/Universal_Robots_ROS2_GZ_Simulation/blob/ros2/ur_simulation_gz/urdf/ur_gz.urdf.xacro) assembling the kinematic description and the control description. We made those changes on purpose in order to remove dependencies and move the level of concern to where it actually belongs.

**Comment by MatthijsBurgh** [comment]
That is fine. It means the rviz config is not always in the description pkg.

So do you want to create another argument for that? And maybe also for the rviz config filename?

**Comment by fmauch** [comment]
I would say, let's not worry about that in this PR, that is what we have #22 for.

**Comment by MatthijsBurgh** [comment]
So any changes needed here or not?

**Comment by fmauch** [comment]
No changes needed here, just the docstrings. Thank you for your continuous effort!

**Comment by MatthijsBurgh** [comment]
But I already included more changes than just the docstrings. So are you happy with the current status of this PR?

**Comment by fmauch** [comment]
Sorry, my statement was not clear. Please remove all the changes that are not related to the docstrings. This PR should only alter the docstrings that are present on the current `ros2` branch.

**Comment by MatthijsBurgh** [comment]
@fmauch done

**Comment by fmauch** [comment]
Thank you @MatthijsBurgh It seems like the `prefix` argument of the `ur_sim_moveit.launch.py` file has slipped through. Apart from that this is looking good.

**Comment by MatthijsBurgh** [comment]
@fmauch sorry, I missed that last argument and your comment. Should be fixed now.

**Review by fmauch — APPROVED** [review]
Thanks for coming back to this @MatthijsBurgh, this looks good!

**Pull request merged**